caps/forms.py

```diff
@@ -26,9 +31,22 @@ def search(self):
         else:
             sqs = super(HighlightedSearchForm, self).search()
 
+        if self.cleaned_data["similar_type"]:
+            council_slug = self.cleaned_data["similar_council"]
+            comparison_slug = self.cleaned_data["similar_type"]
+            cut_off = 15
+            names = (Distance
+                     .objects
+                     .filter(council_a__slug=council_slug,
+                             type__slug=comparison_slug,
+                             position__lte=cut_off)
+                     .values_list("council_b__name", flat=True))
+            
+            sqs = sqs.filter(council_name__in=names)
```

Comment:
Using filter means we include the council name in the highlights which I don't think is what we want. This seems to do the same:
```
query_names = [ 'council_name:"{}"'.format(name) for name in names ]
            sqs = sqs.narrow(" OR ".join(query_names))
```

Reply:
That makes sense. Have done it with an f-string, but that works fine.